<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>437</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Save">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>330</y>
      <width>72</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="View">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>330</y>
      <width>72</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>View</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>75</x>
      <y>35</y>
      <width>74</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
      <pointsize>17</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Table List</string>
    </property>
   </widget>
   <widget class="QListWidget" name="TableList">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>351</width>
      <height>192</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Select">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>270</y>
      <width>82</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="bodycare">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>70</y>
      <width>70</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.Apple SD Gothic NeoI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>바디케어</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cleansing">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>150</y>
      <width>59</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.Apple SD Gothic NeoI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>클렌징</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="makeup">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>110</y>
      <width>70</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.Apple SD Gothic NeoI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>메이크업</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="skincare">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>70</y>
      <width>70</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.Apple SD Gothic NeoI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>스킨케어</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="beauty_tool">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>230</y>
      <width>59</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.Apple SD Gothic NeoI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>뷰티툴</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="fragrance">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>230</y>
      <width>82</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.Apple SD Gothic NeoI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>프래그런스</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="menscare">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>150</y>
      <width>70</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.Apple SD Gothic NeoI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>맨즈케어</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="haircare">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>110</y>
      <width>70</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.Apple SD Gothic NeoI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>헤어케어</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="suncare">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>190</y>
      <width>59</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.Apple SD Gothic NeoI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>선케어</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="maskpack">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>190</y>
      <width>75</width>
      <height>18</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.Apple SD Gothic NeoI</family>
      <pointsize>13</pointsize>
     </font>
    </property>
    <property name="text">
     <string>마스크/팩</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Dup_check">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>270</y>
      <width>142</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
     </font>
    </property>
    <property name="text">
     <string>Duplicate Check</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Status">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>330</y>
      <width>81</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
     </font>
    </property>
    <property name="text">
     <string>Status</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Upload">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>330</y>
      <width>86</width>
      <height>32</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>.AppleSystemUIFont</family>
     </font>
    </property>
    <property name="text">
     <string>Upload</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
